pkg/eks/services_v2.go

```diff
@@ -11,6 +11,7 @@ import (
 	"github.com/aws/aws-sdk-go-v2/service/ssm"
 	"github.com/aws/aws-sdk-go-v2/service/sts"
 
+	"github.com/weaveworks/eksctl/pkg/apis/eksctl.io/v1alpha5"
```

Comment:
The convention is to use the `api` alias for this package. 

```suggestion
	api "github.com/weaveworks/eksctl/pkg/apis/eksctl.io/v1alpha5"
```